src/services/huggingface-service.ts

```
@@ -148,40 +160,41 @@
-      
+      await execAsync(`huggingface-cli download ${huggingfaceId} --local-dir "${modelPath}"`, {
+        timeout: this.config.timeout,
+      });
+
       onProgress?.(`Model downloaded to ${modelPath}`);
       return modelPath;
     } catch (error) {
       // Clean up partial download
       try {
         await fs.rm(modelPath, { recursive: true, force: true });
       } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)